ci/cppcheck/cppcheck-new-issues.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
-scriptdir="$(cd "${0%/*}" && pwd)"
-rootdir="${scriptdir%/*/*}"
+scriptdir=$(dirname "$(readlink -f "$0")")
+rootdir=$(realpath "$scriptdir/../..")
 
-# shellcheck source=../../tools/functions.sh
+# shellcheck source=tools/functions.sh
+# shellcheck disable=SC1091
```

Comment:
No no no - make sure you include functions.sh in your shellcheck invocation and this won't be needed.

Reply:
Reverted my changes.